README.md

```diff
@@ -1,5 +1,51 @@
-# AUMS-api
-an API endpoint for AUMS ( Amrita University Management System ) made using Node.js and Express
+# Amrita University Management system API 
 
-hosted at : https://aumshelper.herokuapp.com/api/aums
+[![Gopala Krishnan| Amrita](https://cldup.com/dTxpPi9lDf.thumb.png)](https://www.amrita.edu/)
+
+AUMS-api is an API built to ease the developement process for native/web apps working on top of it. The author has built the API using Express and Node. The api is currently shifting from its developement phase to deployment, with features to be rolled out in the near future. The readme is the broad overlay of the following
+
+  - Getting started with requests in A-api
+  - Developement and Contribution
+
+# Getting started with AUMS-api!
+
+The api has been hosted on Heroku thanks to its 750 free hours of computation and its flawless working with simple applications. The following lines will focus on sending POST requests to the api for simple queries and a more detailed procedure on a select few. The API currently supports 7 POST requests with more scheduled for later. Down below we post a request to access student-grade records. 
+
+ # Note
+> The author recommends sending cookies from session in the post request for faster returns. All requests are maintained so as to return the session cache for every query along with the requested attributes.
+
+We send the appropriate request from heroku as follows:
+```sh
```

Comment:
Define for each and every route, the parameters required and how the response is given by the API. Also write the URL's